ODS box fails to build

**Describe the bug**
Latest execution of e2e tests in ods are failing due to an error in one mirror ([packages.endpoint.com](https://packages.endpoint.com/rhel/7/os/x86_64/endpoint-repo-1.7-1.x86_64.rpm))

**To Reproduce**
Steps to reproduce the behavior:
1. Create an ODS-box 
4. See error

**Expected behavior**
ODS-box should be created successfully

**Affected version (please complete the following information):**
 - OpenShift: 3.11
 - OpenDevStack 3.x, 4.x, master

**Log Output**
```
[2022-02-02T12:13:40.364Z] [0;32m    amazon-ebs: Loaded plugins: fastestmirror, langpacks[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Loading mirror speeds from cached hostfile[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:  * base: download.cf.centos.org[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:  * extras: download.cf.centos.org[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:  * updates: download.cf.centos.org[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Package yum-utils-1.1.31-54.el7_8.noarch already installed and latest version[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Examining /var/tmp/yum-root-tPkBLc/ius-release-el7.rpm: ius-release-2-1.el7.ius.noarch[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Marking /var/tmp/yum-root-tPkBLc/ius-release-el7.rpm to be installed[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Resolving Dependencies[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: --> Running transaction check[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: ---> Package epel-release.noarch 0:7-11 will be installed[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: ---> Package ius-release.noarch 0:2-1.el7.ius will be installed[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: --> Finished Dependency Resolution[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Dependencies Resolved[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: ================================================================================[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:  Package            Arch         Version           Repository              Size[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: ================================================================================[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Installing:[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:  epel-release       noarch       7-11              extras                  15 k[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:  ius-release        noarch       2-1.el7.ius       /ius-release-el7       4.5 k[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Transaction Summary[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: ================================================================================[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Install  2 Packages[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Total size: 19 k[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Total download size: 15 k[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Installed size: 29 k[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Downloading packages:[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Running transaction check[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Running transaction test[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Transaction test succeeded[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Running transaction[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:   Installing : epel-release-7-11.noarch                                     1/2[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:   Installing : ius-release-2-1.el7.ius.noarch                               2/2[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:   Verifying  : ius-release-2-1.el7.ius.noarch                               1/2[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:   Verifying  : epel-release-7-11.noarch                                     2/2[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Installed:[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:   epel-release.noarch 0:7-11          ius-release.noarch 0:2-1.el7.ius[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs:[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Complete![0m
[2022-02-02T12:13:40.365Z] [1;31m==> amazon-ebs: + sudo yum -y install https://packages.endpoint.com/rhel/7/os/x86_64/endpoint-repo-1.7-1.x86_64.rpm[0m
[2022-02-02T12:13:40.365Z] [0;32m    amazon-ebs: Loaded plugins: fastestmirror, langpacks[0m
[2022-02-02T12:13:40.365Z] [1;31m==> amazon-ebs: Cannot open: https://packages.endpoint.com/rhel/7/os/x86_64/endpoint-repo-1.7-1.x86_64.rpm. Skipping.[0m
[2022-02-02T12:13:40.365Z] [1;31m==> amazon-ebs: Error: Nothing to do[0m
[2022-02-02T12:13:40.365Z] [1;31m==> amazon-ebs: Script exited with non-zero exit status: 1.Allowed exit codes are: [0][0m
[2022-02-02T12:13:40.365Z] 2022/02/02 12:13:36 [INFO] 641046 bytes written for 'stdout'
[2022-02-02T12:13:40.365Z] 2022/02/02 12:13:36 packer-builder-amazon-ebs plugin: [ERROR] Remote command exited with '1': /tmp/script_5112.sh  --branch master
```
@stitakis

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -52,6 +52,7 @@
 - Update jenkins plugins ([#1040](https://github.com/opendevstack/ods-core/pull/1040))
 - Do not replace multiple occurences of project in component name ([#1078](https://github.com/opendevstack/ods-core/issues/1078))
 - Jenkins Agent Base UBI8 fix new Centos repos ([#1093](https://github.com/opendevstack/ods-core/pull/1093))
+- Update centos mirror ([#1098](https://github.com/opendevstack/ods-core/pull/1098))
 
 ### Removed
 

--- a/ods-devenv/scripts/bootstrap.sh
+++ b/ods-devenv/scripts/bootstrap.sh
@@ -17,7 +17,7 @@ echo "bootstrap: Will build ods box against git-ref ${ods_git_ref}"
 if [[ -n $(command -v git) ]]; then sudo yum remove -y git*; fi
 sudo yum update -y
 sudo yum install -y yum-utils epel-release https://repo.ius.io/ius-release-el7.rpm
-sudo yum -y install https://packages.endpoint.com/rhel/7/os/x86_64/endpoint-repo-1.7-1.x86_64.rpm
+sudo yum -y install https://packages.endpointdev.com/rhel/7/os/x86_64/endpoint-repo.x86_64.rpm
 sudo yum -y install git
 
 opendevstack_dir="${HOME}/opendevstack"

